content/en/blog/_posts/grokkin-the-docs.md

```diff
@@ -0,0 +1,327 @@
+
+Sometimes the review process made me feel defensive, annoyed, and frustrated. I'm
+sure other contributors have felt the same way. Contributors need to be patient!
+Writing excellent documentation is an iterative process. Reviewers scrutinize
+PRs because they want to maintain a high level of quality in the documentation,
+not because they want to annoy contributors!
+
+## Observation 06: Make every word count {#observation-6}
+
+Non-native English speakers read and contribute to the Kubernetes documentation.
+When you are writing content, use simple, direct language in clear, concise
+sentences.  Every sentence you write may be translated into another language, so
+remove words that don't add substance. I admit that implementing these
+guidelines is challenging at times.
+
+Issues and pull requests aren't translated into other languages. However, you
+should still follow the aforementioned guidelines when you write the description
+for an issue or pull request. You should add details and background
+information to an issue so the person doing triage doesn't have to apply the
+`triage/needs-information` label. Likewise, when you create a pull request, you
+should add enough information about the content change that reviewers don't have
+to figure out the reason for the pull request. Providing details in clear,
+concise language speeds up the process.
+
+## Observation 07: Triaging issues is more difficult than it should be {#observation-7}
+
+In SIG Docs, triaging issues requires the ability to distinguish between
+support, bug, and feature requests not only for the documentation but also for
+Kubernetes code projects. How to route, label, and prioritize issues has become
+easier week by week. I'm still not 100% clear on which SIG and/or project is
+responsible for which parts of the documentation. The SIGs and Working Groups
+[page](https://github.com/kubernetes/community/blob/master/sig-list.md) helps,
+but it is not enough. At a page level in the documentation, it's not
+always obvious which SIG or project has domain expertise. The page's front
+matter sometimes list reviewers but never lists a SIG or project. Each page should
+indicate who is responsible for content, so that SIG Docs triagers know where to
+route issues.
+
+## Observation 08: SIG Docs is understaffed {#observation-8}
+
+Documentation is the #1 driver of software adoption<sup>1</sup>.
+
+Yet few companies are willing to allocate tech writers at least
+half-time to Kubernetes. Many contributors are able to devote a small amount of
+time to SIG Docs but few are full-time. Of the people __reviewing and
+approving__ English language documentation, probably only one or two are
+allocated full-time. Of the people __contributing content__ to the English
+language documentation, probably less than a handful are allocated full-time.
+That's very disheartening for online documentation that has had over 53 million
+unique page views from users in 229 countries year-to-date in 2019.  
```

Comment:
I 💔 hyphens.

```suggestion
unique page views from users in 229 countries year to date in 2019.  
```

Reply:
Either is correct, but I bow to the wandering editor's red pen.  lol